cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/Executor.java

```diff
@@ -463,6 +464,7 @@ public void run() {
     private void execute() {
       _state = ExecutorState.State.STARTING_EXECUTION;
       _executorState = ExecutorState.executionStarted(_uuid, _recentlyDemotedBrokers, _recentlyRemovedBrokers);
+      RESULT_LOG.info(String.format("Proposal execution:[%s] started.", _uuid));
```

Comment:
Would it be possible to go over all `RESULT_LOG.info(...)` to ensure that we use lazy eval?